components/ILIAS/UICore/classes/Screen/PageContentProvider.php

```diff
@@ -173,7 +173,7 @@ private function buildTabTitle(): string
 
     public function getFooterModification(CalledContexts $screen_context_stack): ?FooterModification
     {
-        return $this->globalScreen()->layout()->factory()->footer()->withModification(fn(?Footer $footer): ?Footer => $this->dic['legalDocuments']->modifyFooter($footer));
+        return $this->globalScreen()->layout()->factory()->footer();
```

Comment:
return `null` or remove the whole method `getFooterModification` would be better here.

Reply:
You are right. I removed the method.